Caltech-IPAC/firefly
# Firefly-91: Add centering drop down menu

Firefly-91: Add centering drop down menu

 - Center on target
 - Center on image
 - Add drop down
 - Modified for brief target panel / cleaned up target panel
 - Added recent cascade
 - CompleteButton will now optionally respond to enter
 - 10 Recent targets saved as a preference
 - added start up option support
 - added watcher to center image on table row

_to test:_

https://irsawebdev9.ipac.caltech.edu/firefly-91-center-dropdown/firefly/

- Load a HiPS (or image but HiPs makes for a better test)
  - Goto the Center dropdown ![center-dropdown](https://user-images.githubusercontent.com/6730519/60361891-ab560500-999c-11e9-9d28-578509effeb9.png)
  - Try Center on target (will be enable if you entered a target)
  - click on the position input and try positions, watch HiPS move
  - look to see if it saves your recent positions
- Load two catalogs from two different positions
  - click on various rows of the table and see the hips move to the position
